llvm/test/CodeGen/DirectX/clean-module-flags.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: opt -S -dxil-prepare < %s | FileCheck %s
+
+; Make sure behavior flag > 6 is fixed.
+; CHECK-NOT: !{i32 7, !"frame-pointer", i32 2}
```

Comment:
Interested in why:

* no CHECK line for wchar_size?
* CHECK-NOT is used for frame-pointer, but CHECK is used for Dwarf Version

Reply:
For DXIL, it is legal to not have wchar_size and frame-pointer.
But remove Dwarf Version will lost information for DXIL.

Reply:
This check doesn't seem correct. The code change here doesn't remove the flag it changes the behavior value. We should check for the updated behavior not check that the old behavior doesn't exist.